retoolsApp/src/retools.cpp

```diff
@@ -132,6 +131,34 @@ long epicsShareAPI reAddInfo(const char *pattern, const char *name,
         });
 }
 
+long epicsShareAPI rePutField(const char *pattern, const char *field,
+        const char *value)
+{
+    if (!pattern || !field || !value) {
+        errlogSevPrintf(errlogMinor,
+                "Usage: %s \"pattern\" \"field\" \"value\"\n", __func__);
+        return EXIT_FAILURE;
+    }
+
+    return forEachMatchingRecord(pattern, value,
+        [field](DBENTRY *entry, string const & recName, string const & value) {
+	    DBADDR addr;
+	    string fieldName = recName + "." + field;
+	    
+	    if(dbNameToAddr(fieldName.c_str(), &addr))
```

Comment:
Hi @nariox one last thing! Can you fix the indentation of these lines? Everything else looks good and I will merge as soon as this little thing is fixed. Thanks again for the contribution!

Reply:
Sure thing!

Your BIM pals send their regards!